src/Action/ExceptionAction.php

```diff
@@ -27,20 +25,15 @@
  */
 final class ExceptionAction
 {
-    const DEFAULT_EXCEPTION_TO_STATUS = [
-        ExceptionInterface::class => Response::HTTP_BAD_REQUEST,
-        InvalidArgumentException::class => Response::HTTP_BAD_REQUEST,
-    ];
-
     private $serializer;
     private $errorFormats;
     private $exceptionToStatus;
 
-    public function __construct(SerializerInterface $serializer, array $errorFormats, $exceptionToStatus = [])
+    public function __construct(SerializerInterface $serializer, array $errorFormats, array $exceptionToStatus = [])
```

Comment:
I would add a phpdoc with examples of the possible values for `$errorFormats` and `$exceptionToStatus`, it is not clear from just looking at the constructor what those values can be although the tests gives some hints